webdriver/tests/bidi/browsing_context/context_created/context_created.py

```diff
@@ -136,19 +136,21 @@ async def on_event(method, data):
     children_info = root_info["children"]
     assert len(children_info) == 2
 
+    # Note: `browsingContext.contextCreated` is always created with "about:blank":
+    # https://github.com/w3c/webdriver-bidi/issues/220#issuecomment-1145785349
```

Comment:
According to our conversation during the last meeting, only "live" contextCreated events will be created with "about:blank". Cached events should be emitted with the current URL. 

```suggestion
    # Note: Live `browsingContext.contextCreated` events are always created with "about:blank":
    # https://github.com/w3c/webdriver-bidi/issues/220#issuecomment-1145785349
```